__docs__/wonder-blocks-styles/__overview__.mdx

```diff
@@ -0,0 +1,31 @@
+import {Meta} from "@storybook/blocks";
+
+import Banner from "@khanacademy/wonder-blocks-banner";
+import {View} from "@khanacademy/wonder-blocks-core";
+import {spacing} from "@khanacademy/wonder-blocks-tokens";
+
+<Meta title="Packages / Styles / Overview" />
+
+# Wonder Blocks Styles
+
+Wonder Blocks Styles provides a set of styles that can be used to style your
+components. These styles are defined in the `@khanacademy/wonder-blocks-styles`
+package.
+
+These styles are meant to be used as global patterns for styling your
+components. They are not meant to be used as a replacement for your own
+component styles.
```

Comment:
What does this mean? Should developers not use these styles directly?

Reply:
hmm good question, I'll reword it to state that we still suggest using WB components as the default solution (instead of "your own component styles"). Do you think that would make more sense? or there would be missing more clarification?

Reply:
My understanding is that dev teams should prioritize using WB components and then use these global styles (and tokens!) when creating custom components that aren't in the design system!